aspnetcore/client-side/blazor/introduction/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Introduction to Blazor
+author: guardrex
+description: Learn how Blazor runs in the browser to execute C#/Razor code with WebAssembly and the Mono runtime in this introduction.
+manager: wpickett
+ms.author: riande
+ms.custom: mvc
+ms.date: 03/26/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: client-side/blazor/introduction/index
+---
+# Introduction to Blazor
+
+By [Steve Sanderson](http://blog.stevensanderson.com), [Daniel Roth](https://github.com/danroth27), and [Luke Latham](https://github.com/guardrex)
+
+[!INCLUDE[](~/includes/blazor-preview-notice.md)]
+
+Blazor is a new experimental .NET web framework using C#/Razor and HTML that runs in the browser with [WebAssembly](http://webassembly.org). Blazor provides all of the benefits of a rich client-side web UI framework using .NET on both the server and the client.
```

Comment:
~~Blazor is a new experimental~~
Blazor is an experimental

- Add a comma between "rich" and "client-side".

Reply:
Let's drop "rich." The main message of the sentence isn't served by it.